File: TD/TD2/Code/2.cpp
/* Calculer la somme des N premiers nombres impairs */
#include <iostream>

int main()
{
  int a;
  int b;
  
  b = 0;
  
  std::cout << "Donne moi une valeur : " << std::flush;
  std::cin >> a;
  
  for (int i = 1; i < 2 * a; i += 2) {
    b += i;
  }
  
  std::cout << b << std::endl;
  
  return 0;
}

File: TD/TD2/Code/Python/2.py
#!/usr/bin/python
# Calculer la somme des N premiers nombres impairs
somme = 0

print('Donne moi une valeur : ', end='')
a = int(input())

for i in range(1, a * 2, 2):
    somme += i

print('La somme est égal à', somme)
